impl/client/src/main/javascript/web/pentaho/data/filter/abstract.js

```diff
@@ -429,6 +429,59 @@ define([
         return result;
       },
 
+      /**
+       * Gets the [extensional]{@link https://en.wikipedia.org/wiki/Extensional_definition} representation of a [Filter]{@link pentaho.data.filter.spec.IFilter}.
+       *
+       * @param {!pentaho.data.AbstractTable} dataTable
```

Comment:
Parameters need descriptions (at least in public methods:
`@param {!pentaho.data.ITable} dataTable - The data table on which to evaluate the extension.`

Reply:
Should not reveal the `AbstractTable` class, if possible